x-pack/functionbeat/provider/aws/aws/transformer/transformer.go

```diff
@@ -210,3 +218,82 @@ func SQS(request events.SQSEvent) []beat.Event {
 	}
 	return events
 }
+
+func S3GetEvents(request events.S3Event) ([]beat.Event, error) {
+	var evts []beat.Event
+	svc := s3.New(session.New())
+	for _, record := range request.Records {
+		unescaped_key, err := url.QueryUnescape(record.S3.Object.Key)
+
+		if err != nil {
+			fmt.Println("Got error unescaping key: %s", record.S3.Object.Key)
```

Comment:
Please remove all `fmt.Println` lines from your PR. We only return the errors from functions and let the callers/Beats take care of printing the error to the log.

Reply:
done ;)